tests_e2e/test_journey.py

```diff
@@ -0,0 +1,68 @@
+from time import sleep
+from selenium.webdriver.common.by import By
+import pytest 
+
+def test_check_title(driver, test_app):
+    driver.get('http://localhost:5000/')
+    assert driver.title == 'To-Do App'
+
+def test_add_todo_item(driver, test_app):
+    driver.get('http://localhost:5000/')
+    text = "Test 1 To Do Item"
+
+    populateTitleInputElement(driver, text)
+    submitTitleButtonElement(driver)
+
+    newToDoItemXPath = '/html/body/div/div[2]/div[2]/div/ul[1]/div/li/ul/div/div[1]'
```

Comment:
Could you simplify these XPath commands (ideally by adding classes in the HTML?)